src/filesystem/interface.js

```diff
@@ -1,67 +1,70 @@
-var { promisify } = require('es6-promisify');
-
-var Path = require('../path.js');
-var nop = require('../shared.js').nop;
-
-var Constants = require('../constants.js');
-var FILE_SYSTEM_NAME = Constants.FILE_SYSTEM_NAME;
-var FS_FORMAT = Constants.FS_FORMAT;
-var FS_READY = Constants.FS_READY;
-var FS_PENDING = Constants.FS_PENDING;
-var FS_ERROR = Constants.FS_ERROR;
-var FS_NODUPEIDCHECK = Constants.FS_NODUPEIDCHECK;
-
-var providers = require('../providers/index.js');
-
-var Shell = require('../shell/shell.js');
-var Intercom = require('../../lib/intercom.js');
-var FSWatcher = require('../fs-watcher.js');
-var Errors = require('../errors.js');
-var defaultGuidFn = require('../shared.js').guid;
-
-var STDIN = Constants.STDIN;
-var STDOUT = Constants.STDOUT;
-var STDERR = Constants.STDERR;
+'use strict';
+
+const { promisify } = require('es6-promisify');
+
+const Path = require('../path.js');
+const nop = require('../shared.js').nop;
```

Comment:
Since we pull two things out of `shared.js` (here and below on line 15), let's do ti all together:

```js
const {
  nop,
  guid: defaultGuidFn
} = require('../shared.js');